.github/workflows/build.yml

```diff
@@ -91,6 +91,7 @@ jobs:
         with:
           php-version: ${{ matrix.php-version }}
           extensions: mbstring, intl, xml
+          ini-values: "error_reporting=E_ALL"
```

Comment:
https://www.php.net/manual/en/errorfunc.configuration.php#ini.error-reporting
Prior to PHP 8.0.0, the default value was: E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED. This means diagnostics of level E_NOTICE, E_STRICT and E_DEPRECATED were not shown.

That's why we need to enable E_ALL in test to get all errors